doc/source/guide/installation/index.rst

```diff
@@ -38,7 +38,7 @@ To install SunPy launch a system command prompt or the
 
 To configure conda for sunpy downloads run::
 
-    conda config --add channels sunpy
+    conda config --add channels sunpy --add channels astropy
```

Comment:
I thought `astropy` was in conda by default...  why do we need to add their channel for?